octoprint_mrbeam/templates/wizard/wizard_whatsnew_0.jinja2

```diff
@@ -1,9 +1,28 @@
+<h3>{{ _("New Software Channels!") }}</h3>
+
+<div id="whatsnew_software_channel_descriptions" class="row">
+    <div class="control-group span6 ">
+        <h5><i class="fa fa-thumbs-o-up" aria-hidden="true"></i>{{ _('Stable Channel') }}: </h5>
+        <p>{{ _('Use the most stable software for your Mr Beam II. You will only get notified about officially released and well tested software updates.') }}</p>
+    </div>
+    <div class="control-group span6">
+        <h5><i class="fa fa-flask" aria-hidden="true"></i>{{ _('Beta Channel') }}: </h5>
+        <p>{{ _('Get the latest Mr Beam development! In this channel you can try the newest features even before they are officially released, with more frequent updates. Use this if you are curious, you can always switch back to the stable channel if you experience errors.') }}</p>
+    </div>
+</div>
+
+<div id="whatsnew_software_channel_choose">
+    <label for="whatsnew_software_channel_select"><strong>{{ _('Select a channel for your Mr Beam II:') }}</strong></label>
+    <select id="whatsnew_software_channel_select">
+        <option value="PROD" selected="selected">Stable</option>
+        <option value="BETA">Beta</option>
+        {% if env_local == 'DEV' %}
+            <option value="DEV">Develop</option>
+        {% endif %}
+    </select>
+    <div id="whatsnew_software_channel_switch">
+        <i class="fa fa-info" aria-hidden="true"></i>
+        {{ _('Switch between the different channels of the Mr Beam II software any time in the settings!') }}
```

Comment:
Please make this line appear in blue color

Reply:
Fixed in f_channel_explanation_settings